src/Symfony/Component/DependencyInjection/Compiler/InlineCompilePass.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\CompilerExtensionInterface;
+
+/**
+ * A pass to execute inline compiler passes in extensions.
+ *
+ * @author Wouter J <wouter@wouterj.nl>
+ */
+class InlineCompilePass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        foreach ($container->getExtensions() as $extension) {
+            if (!$extension instanceof CompilerExtensionInterface) {
```

Comment:
@WouterJ actually, `compile()` is misleading as the container builder is not compiled here. It is still being processed, so I'll vote for `process()`.

Wouldn't it be better to call this class `InlineCompilerPass` or `ExtensionCompilerPass`?